### PR TITLE
[Term Entry] Java Calendar Function: .getActualMinimum()

### PR DESCRIPTION
**Pull request by JJAyala1010**
### Description

-Added getActualMinimum term for Calendar.getActualMinimum() term for Java
-Closes Issue #2975 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md
Comment (CBID2) [inline content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md:26]: [Question/proofreading] Is this statement for the below  code snippet? If so, I highly recommend using this method to clarify this point:
```suggestion
The method below returns the minimum value for a calendar year
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @JJAyala1010! :) Your entry is of to a pretty good start! :) I gave a suggestion on how to improve it. I hope you find my advice helpful! 😊👍🏾

File: content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md
Comment (SSwiniarski) [inline content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md:3]:
```suggestion
Description: 'Returns the minimum value that the specified calendar field could have based on the time value of the calendar.'
```

File: content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md
Comment (SSwiniarski) [inline content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md:9]:
```suggestion
  - 'Date'
  - 'Time'
```
Need to use only tags from this list: https://github.com/Codecademy/docs/blob/main/documentation/tags.md

File: content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md
Comment (SSwiniarski) [inline content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md:15]:
```suggestion
The **`.getActualMinimum()`** method returns the minimum possible value that the specified calendar field can have. 
```

File: content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md
Comment (SSwiniarski) [inline content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md:48]:
```suggestion
```

File: content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md
Comment (SSwiniarski) [inline content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md:17]:
```suggestion
## Syntax

```
blank line after header

File: content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md
Comment (SSwiniarski) [inline content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md:23]:
```suggestion
min = myCalendar.getActualMinimum(field);
```
The syntax block should be more general, and just illustrating the method in use.

File: content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md
Comment (SSwiniarski) [inline content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md:26]:
```suggestion
Where `myCalendar` is a `Calendar` object, and `field` is the constant for the field to retrieve the minimum for.
```
This sentence should be describing elements of the syntax example.

File: content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md
Comment (SSwiniarski) [inline content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md:30]: Before the example should be an introductory sentence summarizing what the example is doing.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@JJAyala1010 I did a first maintainer review and left a few comments. Note my comments on the syntax section override @CBID2's comment.

File: content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md
Comment (SSwiniarski) [inline content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md:9]:
```suggestion
  - 'Calendar'
  - 'Time'
  - 'Date'
```

Comment (JJAyala1010) [reply content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md:9]: Thank you for the suggestions. I've made and committed the changes. Let me know if I did something wrong or if the changes don't show for some reason.

File: content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md
Comment (SSwiniarski) [inline content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md:20]:
````suggestion
```pseudo
min = cal.getActualMinimum(field);
```
````

File: content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md
Comment (SSwiniarski) [inline content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md:28]:
```suggestion
## Example

The below example uses getActualMinimum() to retrieve the minimum value for a year and the minimum value for a month.
```

File: content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md
Comment (SSwiniarski) [inline content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md:47]:
```suggestion
```
Remove extra blank lines.

**Review by SSwiniarski — COMMENTED** [review]
@JJAyala1010 A few more comments.

File: content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md
Comment (JJAyala1010) [reply content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md:21]: Have added these new suggestions now

**Review by SSwiniarski — APPROVED** [review]
@JJAyala1010 I did one minor edit. Looks good for a second review.

File: content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md:3]: This description uses the same wording as the definitions on other sites-
https://www.tutorialspoint.com/java/util/calendar_getactualminimum.htm
https://www.geeksforgeeks.org/calendar-getactualminimum-method-in-java-with-examples/

There is no plagiarism permitted in docs, all content must be original. Please update the description using your own words. Also update line 16 to be in alignment with the new description.

File: content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md:23]:
```suggestion
Where `myCalendar` is a `Calendar` object, and `field` is the constant for which the minimum will be returned.
```

File: content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md:27]:
```suggestion
The below example uses `.getActualMinimum()` to retrieve the minimum value for a year and the minimum value for a month.
```

File: content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md:38]:
```suggestion
    // Return the minimum value that the year field could have
```

File: content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md:40]:
```suggestion
    System.out.println("Minimum year: " + a);
```

File: content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md:44]:
```suggestion
    System.out.println("Minimum month: " + i);
```

File: content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md:42]:
```suggestion
    // Return the minimum value that the month field could have
```

File: content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md
Comment (caupolicandiaz) [inline content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md:53]:
```suggestion
Minimum year: 1
Minimum month: 0
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@JJAyala1010 I've added my initial comments. Please review carefully, and make the necessary updates. Tag me when you've finished making changes. Thanks

**Comment by caupolicandiaz** [comment]
@JJAyala1010 it has been several weeks are you still working on this entry?

**Comment by SSwiniarski** [comment]
@caupolicandiaz In the interest of getting this moving, I implemented all your comments. How does it look now?

**Review by caupolicandiaz — APPROVED** [review]
@JJAyala1010 @SSwiniarski LGTM

**Pull request merged**
